application/services/manifest.py

```
@@ -226,37 +239,107 @@ def get_stage_info(self, stage_id: str) -> Optional[Dict]:
+                "current_stage_spec": current_stage_spec,
+                "collected_info": {
+                    "decisions": decisions,
+                    "artifacts": artifacts,
+                },
+            }
+
+            # 更新上下文
+            context_manager.update_context(stage_id, updates)
+            print(f"  ✓ 上下文已更新: {stage_id}")
+
+        except Exception as e:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)